File: sensor/src/main/java/com/tfm/microservices/sensor/repository/Sensor.java
package com.tfm.microservices.sensor.repository;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
@Data
public class Sensor {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    private String sensorId;
    private String owner;
    private String catastralReference;
    private double latitude;
    private double longitude;
    private String type;
}

File: tfm-front-end/src/app/sensor.ts
export class Sensor {
    sensorId:String;
    catastralReference:String;
    latitude:number;
    longitude:number;
    type:String;

    constructor(sensorId:String, catastralReference:String, latitude:number, longitude:number, type:String){
        this.sensorId = sensorId;
        this.catastralReference = catastralReference;
        this.latitude = latitude;
        this.longitude = longitude;
        this.type = type;
    }
}
